server/game/costs/payxgoldprompt.js

```diff
@@ -0,0 +1,41 @@
+const _ = require('underscore');
+
+const BaseStep = require('../gamesteps/basestep');
+
+class payXGoldPrompt extends BaseStep {
```

Comment:
Class names should always begin with an uppercase letter. I'm surprised there's not a linting rule around this.